api/v1/flow/README.md

```diff
@@ -1049,18 +1049,18 @@ This mirrors enum xlate_point in bpf/lib/trace_sock.h
 | Name | Number | Description |
 | ---- | ------ | ----------- |
 | UNKNOWN_POINT | 0 | Cilium treats 0 as TO_LXC, but its&#39;s something we should work to remove. This is intentionally set as unknown, so proto API can guarantee the observation point is always going to be present on trace events. |
-| TO_PROXY | 1 |  |
-| TO_HOST | 2 |  |
-| TO_STACK | 3 |  |
-| TO_OVERLAY | 4 |  |
-| TO_ENDPOINT | 101 | same as TO_LXC, which had a 0 value. This index is intentionally very high so when new segments are added in bpf, there are no collisions |
-| FROM_ENDPOINT | 5 |  |
-| FROM_PROXY | 6 |  |
-| FROM_HOST | 7 |  |
-| FROM_STACK | 8 |  |
-| FROM_OVERLAY | 9 |  |
-| FROM_NETWORK | 10 |  |
-| TO_NETWORK | 11 |  |
+| TO_PROXY | 1 | TO_PROXY indicates network packets are received by l7 proxy. |
+| TO_HOST | 2 | TO_HOST indicates network packets are received by host namespace. |
+| TO_STACK | 3 | TO_STACK indicates network packets are received by the Linux kernel network stack on host machine. |
+| TO_OVERLAY | 4 | TO_OVERLAY indicates network packets are received by the tunnel device. |
+| TO_ENDPOINT | 101 | TO_ENDPOINT indicates network packets are received by endpoints (containers). |
+| FROM_ENDPOINT | 5 | FROM_ENDPOINT indicates network packets are transmitted from endpoints (containers). |
+| FROM_PROXY | 6 | FROM_PROXY indicates network packets are transmitted from l7 proxy. |
```

Comment:
```suggestion
| FROM_PROXY | 6 | FROM_PROXY indicates network packets were received from the l7 proxy. |
```

Similarly but in reverse, the `FROM_*` targets are from the perspective of Cilium, and Cilium has detected where the packets came from, and one of the first things that Cilium's datapath will do is to emit an event stating that a packet was received from some location outside Cilium's BPF datapath.